File: 1590593189-1676[1]/remoting/Marshalling/By Reference/logClientIIS.vb
Imports System
Imports System.Runtime.Remoting
Imports Wrox4923.Remoting.Log
Imports System.Data
Imports System.Data.SqlClient

Namespace Wrox4923.Remoting

    Public Class logClientIIS

        Public Shared Sub Main() 
			
			RemotingConfiguration.Configure("logClientIIS.exe.config")            					

			Dim httpLogger As New Logger()

            httpLogger.Log("HTTP Client Request")
     	
     	End Sub
    
    End Class

End Namespace

File: 1590593189-1675[1]/remoting/Marshalling/By Reference/logClientIIS.cs
using System;
using System.Runtime.Remoting;
using Wrox4923.Remoting.Log;
using System.Data;

namespace Wrox4923.Remoting
{
    public class logClientIIS
	{
        public static void Main() 
		{	
			RemotingConfiguration.Configure("logClientIIS.exe.config");          					

			Logger httpLogger = new Logger();

            httpLogger.Log("HTTP Client Request");
		}
	}
}